modules/serverless-kafka-sasl-public-certs.adoc

```diff
@@ -0,0 +1,21 @@
+// Module is included in the following assemblies:
+//
+// serverless/event_workflows/serverless-kafka.adoc
+
+[id="serverless-kafka-sasl-public-certs_{context}"]
+= Configuring SASL authentication using public CA certificates
+
+If you want to use SASL with public CA certificates, you must use the `tls.enabled=true` flag, rather than the `ca.crt` argument, when creating the secret. For example:
+
+[source,terminal]
+----
+$ oc create secret --namespace <namespace> generic <kafka_auth_secret> \
+    --from-literal=tls.enabled=true \
+  --from-literal=password="SecretPassword" \
+  --from-literal=saslType="SCRAM-SHA-512" \
+  --from-literal=user="my-sasl-user"
+----
```

Comment:
The 
```
    --from-literal=tls.enabled=true \
```

looks like is having two spaces more than the other arguments

Reply:
Fixed 😄